test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -296,6 +291,9 @@ def verify_category_contains_channels(self, community_category_name, community_c
     
     def open_edit_community_by_community_header(self):
         click_obj_by_name(CommunityScreenComponents.COMMUNITY_HEADER_BUTTON.value)
+        # Without that sleep, the click sometimes lands next to the button and the test fails
+        # The sleep seems to help wait for the page to load and the button to be positioned
+        sleep_test(0.1)
```

Comment:
@StateOf-Vlado Do you know a better way of doing this?
The tests were randomly failing on my machine when this button was involved. Now that I made it smaller, it's always failing without this sleep.

Reply:
It should be sth like `waitForObjectToBeVisible`, in QtTest lingo: https://doc.qt.io/qt-5/qml-qttest-testcase.html#waitForRendering-method

Reply:
> It should be sth like `waitForObjectToBeVisible`, in QtTest lingo: https://doc.qt.io/qt-5/qml-qttest-testcase.html#waitForRendering-method

Will give it a try

Reply:
Something like this seems to do the trick: 

```
verify(is_loaded_visible_and_enabled(CommunitySettingsComponents.EDIT_COMMUNITY_BUTTON.value), "Edit community button is visible and enabled")
```